### ISSUE TITLE
BATCH IMPORT: Large batches don't associate all objects in Version v4.0.0rc1

### ISSUE THREAD

**Issue by Alexandermay** [curationexperts/epigaea#873]
Using Chrome:

1. From import objects select an xml manifest that contains metadata for a large number of objects (In my case 58)

2. Select all of the associated binaries
3. Upload objects
4. A small number of objects will not associate, and MIRA provides a warning message. (See attached.)

**Comment by Alexandermay** [comment]
![Screen Shot 2018-01-11 at 1.49.38 PM.png](https://waffleio-direct-uploads-production.s3.amazonaws.com/uploads/59cd282390570f0012dbecea/125516c66e82c728ace21e0d46db978826878de9bce1b057e40fa0c5710b77226455f26b7191a379f116224f5c575ecc3c4a4356bcbbd671f3e57862804e0ee4d0385f25c23b12f9b8864eea01b821834cd7b3b98818d2f42f67b6d5db1063474f4dcbed5727a19141c95df74ee74453a314376be67f2431.png)

**Comment by Alexandermay** [comment]
Nota Bene: Batch id 121 of 22 objects went in without any of the above issues.

**Comment by no-reply** [comment]
This and #872 look like the same issue as we saw in #785. I'm investigating.

**Comment by no-reply** [comment]
This seems to be an issue with the timing of jquery uploads. I can repeat it pretty reliably by uploading 0 byte files, while large files seem less likely to sneak in before the `fade in` functionality takes over. I think all of the large batch issues will close if we can fix this, but our previous approach seems inadequate.

**Comment by no-reply** [comment]
This is potentially closed by #886. We are going to deploy and confirm.

**Comment by Alexandermay** [comment]
Still doesn't work on my end:
![Screen Shot 2018-02-01 at 7.59.40 AM.png](https://waffleio-direct-uploads-production.s3.amazonaws.com/uploads/59cd282390570f0012dbecea/125516c66e82c728ace21e0d46db978826878de9bce1b057e40fa0c5710b77216454f26b7191a37ff117224f5b5f5edd3c4a4356bcbbd671f3e57862804e0ee4d0385f25c23b12f9b8864eea01b821834cd7b3b98818d2f42f67b6d5db1063474f4dcbed5727a29941c95df74ce54455a5133663e0742431.png)

**Comment by Alexandermay** [comment]
Just tried another set (94) objects and got the same results with the one added wrinkle.  At one point it appears to time out and I get the something went wrong screen.

1. From chrome upload manifest and associate objects.
![Screen Shot 2018-02-01 at 9.01.06 AM.png](https://waffleio-direct-uploads-production.s3.amazonaws.com/uploads/59cd282390570f0012dbecea/125516c66e82c728ace21e0d46db978826878de9bce1b057e40fa0c5710b77216454f26b7191a371f1122a4f5f595edd3c4a4356bcbbd671f3e57862804e0ee4d0385f25c23b12f9b8864eea01b821834cd7b3b98818d2f42f67b6d5db1063474f4dcbed5727a19e41c95df74ce54451a512336ae1732431.png)

2. Associate objects to metadata.  This times out and results in:
![Screen Shot 2018-02-01 at 9.02.35 AM.png](https://waffleio-direct-uploads-production.s3.amazonaws.com/uploads/59cd282390570f0012dbecea/125516c66e82c728ace21e0d46db978826878de9bce1b057e40fa0c5710b77216454f26b7191a371f112294f5c5a5edd3c4a4356bcbbd671f3e57862804e0ee4d0385f25c23b12f9b8864eea01b821834cd7b3b98818d2f42f67b6d5db1063474f4dcbed5727a19041c95df74ce54451a6113161e7712431.png)

3. Return to the home screen, select your batch id and see that not all objects are associated:
![Screen Shot 2018-02-01 at 9.02.52 AM.png](https://waffleio-direct-uploads-production.s3.amazonaws.com/uploads/59cd282390570f0012dbecea/125516c66e82c728ace21e0d46db978826878de9bce1b057e40fa0c5710b77216454f26b7191a371f112294f5a5d5edd3c4a4356bcbbd671f3e57862804e0ee4d0385f25c23b12f9b8864eea01b821834cd7b3b98818d2f42f67b6d5db1063474f4dcbed5727a19041c95df74ce54451ac113467ef712431.png)

**Comment by no-reply** [comment]
I think these are fixed now. See http://epigaea.curationexperts.com/xml_imports/87

I don't believe the timeout issue reported above, and am a bit confused that I can't find the error in Honeybadger. Is there a way to repeat? If so, I'd like to open a new ticket.

**Comment by mark-dce** [comment]
The fix is obscured by the issue described in #895.  We will retest once the fix to #895 is in place.
